modules/social_features/social_branding/src/Wrappers/RGBAColor.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Drupal\social_branding\Wrappers;
+
+use Spatie\Color\Hex;
+use Spatie\Color\Rgba;
+
+/**
+ * A color representation with red, green, blue, and alpha components.
+ */
+class RGBAColor {
```

Comment:
Do we need a separate class for this? Or could we also just have `red`, `green`, `blue` methods on the main `Color` class?